src/Uno.UWP/System/Profile/AnalyticsVersionInfo.Android.cs

```diff
@@ -0,0 +1,64 @@
+#if __ANDROID__
+using System;
+using System.Globalization;
+using Android.App;
+using Android.Content.Res;
+using Android.OS;
+using Uno.Extensions;
+using Uno.Logging;
+using Uno.UI;
+
+namespace Windows.System.Profile
+{
+	public partial class AnalyticsVersionInfo 
+	{
+		public  string DeviceFamily
+		{
+			get
+			{
+				string idiom = "";
+
+				try
+				{
+					// Threshold in inches for diagonal screensize between what HardwareHelper will consider a tablet or a phone.
+					//This is the value given to us by microsoft documentation : https://docs.microsoft.com/en-us/windows/uwp/design/devices/
+					double _tabletMinimumSizeThreshold = 7.0;
+					
+					var displayMetrics = Application.Context.Resources.DisplayMetrics;
```

Comment:
Application.Context may be null, if called to early.

Reply:
@jeromelaban what adjustment would you suggest? We could check if it's null and throw InvalidOperationException saying we're probably calling this too fast?